guides/standards-assurance-tables.md

```diff
@@ -0,0 +1,158 @@
+---
+expires: 2019-09-01
+---
+# Standards Assurance Tables
+
+The MOJ Cyber Security team have developed a 'Standards Assurance Table' (SAT) in the form of a Google Sheet template.
+
+The SAT measures technology systems (and surrounding information governance) against the [UK Cabinet Office](https://www.gov.uk/government/organisations/cabinet-office) [Minimum Cyber Security Standard (MCSS)](https://www.gov.uk/government/publications/the-minimum-cyber-security-standard) and [UK National Cyber Security Centre (NCSC)](https://www.ncsc.gov.uk/) [Cloud Security Principles (CSPs)](https://www.ncsc.gov.uk/collection/cloud-security?curPage=/collection/cloud-security/implementing-the-cloud-security-principles).
```

Comment:
measures -> enables comparison between

Reply:
I don't think it is comparison against, it is measurement for our purposes?